kyuubi-common/src/test/scala/org/apache/kyuubi/operation/IcebergRowLevelOperationTests.scala

```diff
@@ -0,0 +1,103 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.operation
+
+import org.apache.kyuubi.IcebergSuiteMixin
+
+trait IcebergRowLevelOperationTests extends HiveJDBCTestHelper with IcebergSuiteMixin {
+
+  private def createAndInitTable(tableName: String)(records: Seq[(Int, String)]): Unit = {
```

Comment:
we can have a statement as parameter here

Reply:
Do you mean
```
def createAndInitTable(tableName: String)
                      (records: Seq[(Int, String)])
                      (initSql: Option[String] = None): Unit
```

Reply:
private def createAndInitTable(statement: Statement, tableName: String)